sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -192,31 +193,29 @@ static void *_event_loop(void *args)
     return NULL;
 }
 
-#ifdef MODULE_GNRC_SIXLOWPAN
 static void _send_to_iface(kernel_pid_t iface, gnrc_pktsnip_t *pkt)
 {
-    gnrc_ipv6_netif_t *if_entry = gnrc_ipv6_netif_get(iface);
-
     ((gnrc_netif_hdr_t *)pkt->data)->if_pid = iface;
+    gnrc_ipv6_netif_t *if_entry = gnrc_ipv6_netif_get(iface);
 
+    assert(if_entry != NULL);
+    if (gnrc_pkt_len(pkt->next) > if_entry->mtu) {
```

Comment:
I'm not sure why, but maybe the while loop in this function causes the node to hang?